htpy/__init__.py

```diff
@@ -17,9 +19,13 @@
 from htpy._types import Renderable as Renderable
 from htpy._with_children import with_children as with_children
 
-__version__ = "25.5.1"
 __all__: list[str] = []
 
+try:
+    __version__ = version(__name__)
+except PackageNotFoundError:
+    __version__ = "unknown"
```

Comment:
Getting rid of `__version__` altogether would be nice. `importlib.metadata.version` seems like the sensible way to determine the version.